graphql/src/graphql_intf.ml

```diff
@@ -15,6 +15,13 @@ module type IO = sig
   end with type 'a io := 'a t
 end
 
+(* Err signature *)
+module type Err = sig
+  type t
+  val message_of_error : t -> string
+  val extensions_of_error : t -> (string * Yojson.Basic.json [@warning "-3"]) list
```

Comment:
What are your thoughts on making this `t -> (string * Yojson.Basic.json) list option`?

Reply:
Sounds good to me!